pandas/io/json/_json.py

```diff
@@ -439,8 +439,18 @@ def read_json(
            Not applicable for ``orient='table'``.
 
     convert_dates : bool or list of str, default True
-        List of columns to parse for dates. If True, then try to parse
-        datelike columns. A column label is datelike if
+        If True then default datelike columns may be converted.
+        If False, no dates will be converted.
+        If a list of column names, then those columns will be converted and
+        default datelike columns may also be converted
+
+        When True or list then keep_default_dates determines whether default
```

Comment:
I think can just delete this line - duplicate of the one preceding it (?)

Reply:
Thanks for the feedback. I'm happy to reword things (it could probably be more concise)

I think it is important to reference the keep_default_dates parameter, otherwise someone unfamilliar with the functionality would need to look at all parameters in order to find the co-parameter. They may easily miss the significance of the word 'may' and not even look for a co-parameter.

How about changing the description to:

If True then default datelike columns may be converted (depending on keep_default_dates).
If False, no dates will be converted.
If a list of column names, then those columns will be converted and
default datelike columns may also be converted (depending on keep_default_dates).

Reply:
Yep that looks good to me

Reply:
Updated :-)